icons/vitejs/vite-original.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 121.4 119.8" style="enable-background:new 0 0 121.4 119.8;" xml:space="preserve">

```

Comment:
make sure the viewbox is `0 0 128 128` as explained in the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)